module-1/lab-numpy/your-code/katas.py

```diff
@@ -0,0 +1,78 @@
+# codewars
+
+# https://www.codewars.com/kata/insert-dashes
+'''
+Write a function insertDash(num)/InsertDash(int num) that will insert dashes ('-') between each 
+two odd numbers in num. For example: if num is 454793 the output should be 4547-9-3. Don't count 
+zero as an odd number.
+
+insert_dash(454793) ->'4547-9-3'
+
+def list_in_to_string(lista): 
+    res = ""
+    for x in lista: 
+        res += str(x)
+    return res
+'''
+
+def insert_if_needed(n1, n2): 
+    res = False
+    if n1%2 != 0 and n2%2 != 0: 
+        res = True
+    return res    
+
+def insert_dash(num):
+    res = []
+    cad = str(num)
+    for i in range(len(cad)-1): 
+        res.append(cad[i])
+        if insert_if_needed(int(cad[i]), int(cad[i+1])): 
+            res.append('-')
+    res.append(cad[-1]) 
+    print(res)
+    return ''.join(res)
+
```

Comment:
If I call the function insert_dash(454793), do we get that solution? '4547-9-3'
Include the output of each function in your katas file.
I mean for the second Kata 
color_probability('red', 'bumpy') should return the string '0.57'. 
And the same for the third kata. Do you know what I mean?